src/Security/LocalUserProvider.php

```diff
@@ -69,21 +63,36 @@ public function loadUserByToken(OidcToken $token)
         $user->setFamilyName($token->getFamilyName());
         $user->setEmail($token->getEmail());
         $this->em->flush();
+    }
+
+    public function loadOidcUser(string $userIdentifier): UserInterface
```

Comment:
Do we have hybrid account in Kiwi? Oidc and local user should be seperate.

Reply:
We have been having hybrid accounts for some time now. Although I agree that separation might improve the codebase, I feel it's outside the scope of a PR to update the dependencies.

Reply:
I agree that this separation should be it's own issue. for now the priority is updating the new versions